src/server/scripts/Northrend/Ulduar/HallsOfLightning/boss_ionar.cpp

```diff
@@ -87,10 +87,10 @@ class boss_ionar : public CreatureScript
             bIsSplitPhase = true;
             bHasDispersed = false;
 
-            uiSplitTimer = 25 * IN_MILLISECONDS;
+            uiSplitTimer = 25s;
```

Comment:
Instead of using this, you would basically update events or tasks in UpdateAI. And then, schedule these events.